package.json

```diff
@@ -1,7 +1,15 @@
 {
   "private": true,
   "scripts": {
-    "bump-version": "node tools/bump_version.js"
+    "bump-version": "node tools/bump_version.js",
+    "generate": "yarn gen:python && yarn gen:javascript && yarn gen:go && yarn gen:java && yarn gen:kotlin && yarn gen:ruby && yarn gen:csharp",
+    "gen:python": "openapi-generator-cli generate -i openapi.json -g python -o python/ -c python/openapi-generator-config.json -t python/templates",
+    "gen:javascript": "openapi-generator-cli generate -i openapi.json -g typescript -o javascript/src/openapi -c javascript/openapi-generator-config.json",
+    "gen:go": "openapi-generator-cli generate -i openapi.json -g go -o go/internal/openapi -c go/openapi-generator-config.json -t go/templates",
+    "gen:java": "openapi-generator-cli generate -i openapi.json -g java -o java/lib/generated/openapi -c java/openapi-generator-config.json -t java/templates",
+    "gen:kotlin": "openapi-generator-cli generate -i openapi.json -g kotlin -o kotlin/lib/generated/openapi -c kotlin/openapi-generator-config.json -t kotlin/templates",
+    "gen:ruby": "openapi-generator-cli generate -i openapi.json -g ruby -o ruby -c ruby/openapi-generator-config.json -t ruby/templates",
+    "gen:csharp": "openapi-generator-cli generate -i openapi.json -g csharp-netcore -o csharp/ -c csharp/openapi-generator-config.json --global-property apis,models,supportingFiles,apiTests=false,apiDocs=false,modelTests=false,modelDocs=false"
```

Comment:
since you are ignoring these files in the `.openapi-generator-ignore` file is it still necessary to set these global properties to false here?

Reply:
I'm not sure but I'll try messing around with the generator again to verify.